src/main/resources/db/migration/V16__Add_medunderskriverflyt.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE klage.klagebehandling
+    ADD COLUMN medunderskriverflyt_id TEXT;
```

Comment:
Burde man her fyllt feltene med verdier, basert på det som er i db fra før? F.eks så vil vel de som i dag har en medunderskriver satt, egentlig skulle ha SENDT_TIL_MEDUNDERSKRIVER i flyten? For frem til nå så har valg av medunderskriver og sending vært samme operasjon?